### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by iimog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,7 +138,8 @@ The chloroExtractor uses unsortet Fastq files with paired end reads. Please make
 ## Example
 An example data set can be downloaded from zenodo. As example we download the dataset into a folder and run chloroExtractor with the input files.
 
-For preparation, a folder will be created and an example dataset will be downloaded:
+For preparation, a folder will be created and an example dataset will be downloaded.
+If you try this example for your self, either create exactly this folder as well or replace `/tmp/chloroExtractor-testrun` in all commands with the folder that contains the data on your machine.
```
Comment (greatfireball) [inline README.md]: your self --> yourself

Comment (greatfireball) [reply README.md]: ~as well~

File: README.md [hunk 2]
```diff
@@ -133,7 +138,8 @@ The chloroExtractor uses unsortet Fastq files with paired end reads. Please make
 ## Example
 An example data set can be downloaded from zenodo. As example we download the dataset into a folder and run chloroExtractor with the input files.
 
-For preparation, a folder will be created and an example dataset will be downloaded:
+For preparation, a folder will be created and an example dataset will be downloaded.
+If you try this example for yourself, either create exactly this folder as well or replace `/tmp/chloroExtractor-testrun` in all commands with the folder that contains the data on your machine.
```
Comment (greatfireball) [inline README.md]: ~as well~

Comment (greatfireball) [inline README.md]: Should we use a environment variable for the download folder to ensure the same location?

Comment (iimog) [reply README.md]: I like it. Will change accordingly.

**Pull request merged**